<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>YNAB_Sync-Norge</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_Info">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <attribute name="title">
        <string>Info</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="info_stack">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::West</enum>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="elideMode">
           <enum>Qt::ElideRight</enum>
          </property>
          <property name="usesScrollButtons">
           <bool>false</bool>
          </property>
          <property name="documentMode">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <property name="tabBarAutoHide">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="settings_stack">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::West</enum>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="elideMode">
           <enum>Qt::ElideRight</enum>
          </property>
          <property name="usesScrollButtons">
           <bool>false</bool>
          </property>
          <property name="documentMode">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <property name="tabBarAutoHide">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="settings_new">
           <attribute name="title">
            <string>+</string>
           </attribute>
           <attribute name="toolTip">
            <string>Make or import a new settings file.</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="1" column="1">
             <widget class="QPushButton" name="settingsNew_load">
              <property name="text">
               <string>Load (Start)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="settingsNew_import">
              <property name="text">
               <string>Import</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QScrollArea" name="scrollArea">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>745</width>
                 <height>559</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="8" column="0" colspan="2">
                 <widget class="QGroupBox" name="settingsNew_ynab_budget">
                  <property name="title">
                   <string>Budget settings</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_2">
                   <item row="0" column="5">
                    <widget class="QPushButton" name="settingsNew_getBudgets">
                     <property name="text">
                      <string>Hent YNAB Budgets</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1" colspan="5">
                    <widget class="QDateEdit" name="settings_ynab_budget_startdate">
                     <property name="minimumDate">
                      <date>
                       <year>2010</year>
                       <month>1</month>
                       <day>1</day>
                      </date>
                     </property>
                     <property name="displayFormat">
                      <string>yyyy-MM-dd</string>
                     </property>
                     <property name="calendarPopup">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_6">
                     <property name="text">
                      <string>Navn</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>id</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="label_7">
                     <property name="text">
                      <string>start-date</string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="0" colspan="6">
                    <widget class="QTabWidget" name="settings_accountlink">
                     <property name="currentIndex">
                      <number>0</number>
                     </property>
                     <widget class="QWidget" name="settingsNew_newAccountlink">
                      <attribute name="title">
                       <string>+</string>
                      </attribute>
                      <layout class="QGridLayout" name="gridLayout_5">
                       <item row="1" column="1">
                        <widget class="QLabel" name="label_9">
                         <property name="text">
                          <string>SBank kontoer</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <widget class="QLabel" name="label_8">
                         <property name="text">
                          <string>YNAB kontoer</string>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="0" colspan="2">
                        <widget class="QPushButton" name="settingsNew_getAccounts">
                         <property name="text">
                          <string>Hent YNAB og SBanken kontoer</string>
                         </property>
                         <property name="autoDefault">
                          <bool>false</bool>
                         </property>
                         <property name="flat">
                          <bool>false</bool>
                         </property>
                        </widget>
                       </item>
                       <item row="3" column="0" colspan="2">
                        <widget class="QPushButton" name="settingsNew_linkAccounts">
                         <property name="enabled">
                          <bool>false</bool>
                         </property>
                         <property name="text">
                          <string>Link valgte kontoer</string>
                         </property>
                        </widget>
                       </item>
                       <item row="2" column="0">
                        <widget class="QListWidget" name="settings_ynabAccounts">
                         <property name="horizontalScrollBarPolicy">
                          <enum>Qt::ScrollBarAlwaysOff</enum>
                         </property>
                        </widget>
                       </item>
                       <item row="2" column="1">
                        <widget class="QListWidget" name="settings_sbankAccounts">
                         <property name="horizontalScrollBarPolicy">
                          <enum>Qt::ScrollBarAlwaysOff</enum>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </widget>
                   </item>
                   <item row="0" column="1" colspan="4">
                    <widget class="QComboBox" name="settings_ynab_budget_id"/>
                   </item>
                   <item row="1" column="1" colspan="5">
                    <widget class="QLineEdit" name="settings_ynab_budget_name">
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>SBank clientid</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="settings_sbanken_customerid">
                  <property name="text">
                   <string>ddmmyyxxxxx</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>SBank secret</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>YNAB Token</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QLineEdit" name="settings_ynab_token"/>
                </item>
                <item row="0" column="0" colspan="2">
                 <widget class="QLineEdit" name="settings_name">
                  <property name="text">
                   <string>[name]</string>
                  </property>
                  <property name="frame">
                   <bool>false</bool>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Personnr.</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="settings_sbanken_clientid">
                  <property name="text">
                   <string>xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QLineEdit" name="settings_sbanken_secret">
                  <property name="text">
                   <string>********************************************************************</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="1">
                 <widget class="QSpinBox" name="settings_days_back">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="visible">
                   <bool>false</bool>
                  </property>
                  <property name="value">
                   <number>4</number>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QSpinBox" name="settings_waitTime">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="visible">
                   <bool>false</bool>
                  </property>
                  <property name="value">
                   <number>15</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_log">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QListWidget" name="log_list">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
